files/private-chef-cookbooks/private-chef/templates/default/pedant_config.rb.erb

```diff
@@ -175,7 +175,7 @@ ruby_org_acl_endpoint?     false
 ruby_org_assoc?            false
 ruby_organizations_endpoint? false
 chef_12?                   true
-policies?                  <%= node['private_chef']['lb']['xdl_defaults']['policies'] %>
+policies?                  true
```

Comment:
@stevendanna this would have to be set to just false for now if we want to merge this and fix pedant later.